ntbs-service/Pages/Search/Index.cshtml.cs

```diff
@@ -39,7 +51,14 @@ public async Task<IActionResult> OnGetAsync(int? pageIndex)
             IQueryable<Notification> draftsIQ = service.GetBaseQueryableNotificationByStatus(draftStatusList);
             IQueryable<Notification> nonDraftsIQ = service.GetBaseQueryableNotificationByStatus(nonDraftStatusList);
 
-            var notificationsIQ = OrderQueryable(draftsIQ).Union(OrderQueryable(nonDraftsIQ));
+            if (!String.IsNullOrEmpty(IdFilter))
+            {
+                SearchParamsExist = true;

```

Comment:
Do we need this flag? maybe we can use SearchResults to identify it?
If that will take longer you can just use `if (SearchParamsExist = (!String.IsNullOrEmpty(IdFilter)))` as this will assign the value to variable so you don't have to set it later

Reply:
AFAIK it's still necessary as there can still be a search even if no search results are returned

Reply:
Sure sounds good.